packages/core/src/exposed-thing.ts

```diff
@@ -66,19 +66,19 @@ export default class ExposedThing extends TD.Thing implements WoT.ExposedThing {
     };
 
     /** A map of property (read & write) handler callback functions */
-    __propertyHandlers: PropertyHandlerMap = new Map<string, PropertyHandlers>();
+    #propertyHandlers: PropertyHandlerMap = new Map<string, PropertyHandlers>();
```

Comment:
Maybe it is worth to mention that `#` prohibits that the data is serialized into JSON

Reply:
I tried to incorporate this in 2db8506c15a28996b2375c4c60d9ec18e9b1fffc – hope the comments are not too verbose now :/